tests/test_inference.py

```diff
@@ -406,84 +408,25 @@ class TestVariational:
     Tests for tsdate with variational algorithm
     """
 
-    def test_simple_sim_1_tree(self):
-        ts = msprime.simulate(8, mutation_rate=5, random_seed=2)
-        tsdate.date(ts, mutation_rate=5, population_size=1, method="variational_gamma")
-
-    def test_simple_sim_multi_tree(self):
-        ts = msprime.simulate(8, mutation_rate=5, recombination_rate=5, random_seed=2)
-        tsdate.date(ts, mutation_rate=5, population_size=1, method="variational_gamma")
-
-    def test_invalid_priors(self):
-        ts = msprime.simulate(8, mutation_rate=5, recombination_rate=5, random_seed=2)
-        priors = tsdate.prior.MixturePrior(ts, prior_distribution="gamma")
-        grid = priors.make_parameter_grid(population_size=1)
-        grid.grid_data[:] = [1.0, 0.0]  # noninformative prior
-        with pytest.raises(ValueError, match="Non-positive shape/rate"):
-            tsdate.date(
-                ts,
-                mutation_rate=5,
-                method="variational_gamma",
-                priors=grid,
-            )
-
-    def test_custom_priors(self):
-        ts = msprime.simulate(8, mutation_rate=5, recombination_rate=5, random_seed=2)
-        priors = tsdate.prior.MixturePrior(ts, prior_distribution="gamma")
-        grid = priors.make_parameter_grid(population_size=1)
-        grid.grid_data[:] += 1.0
-        tsdate.date(
-            ts,
-            mutation_rate=5,
-            method="variational_gamma",
-            priors=grid,
-        )
-
-    def test_prior_mixture_dim(self):
-        ts = msprime.simulate(8, mutation_rate=5, recombination_rate=5, random_seed=2)
-        priors = tsdate.prior.MixturePrior(ts, prior_distribution="gamma")
-        grid = priors.make_parameter_grid(population_size=1)
-        tsdate.date(
-            ts,
-            mutation_rate=5,
-            method="variational_gamma",
-            priors=grid,
-            prior_mixture_dim=2,
-        )
-
-    def test_bad_arguments(self):
-        ts = utility_functions.two_tree_mutation_ts()
-        with pytest.raises(ValueError, match="Maximum number of EP iterations"):
-            tsdate.date(
-                ts,
-                mutation_rate=5,
-                population_size=1,
-                method="variational_gamma",
-                max_iterations=-1,
-            )
-        with pytest.raises(ValueError, match="must be a nonnegative integer"):
-            tsdate.date(
-                ts,
-                mutation_rate=5,
-                population_size=1,
-                method="variational_gamma",
-                prior_mixture_dim=0.1,
-            )
-
-    def test_match_central_moments(self):
```

Comment:
Should we also leave this in?

Reply:
I think it's fine -- I plan on removing this option as it just causes numerical instability with no improvement in accuracy.

Reply:
... to clarify, we should add tests for various options, but right now i'm swapping them in and out enough that it's a pain. So stuff like this'll get hit in the testing overhaul